src/app/projects/projects.component.html

```diff
@@ -1 +1,125 @@
-<h1>PROJECTS PAGE</h1>
+<div class="container-fluid mt-5">
+	<div class=" container box logos mx-auto d-block">
+		<div class="row text-center">
+			<h1 class="mx-auto mb-5"> OUR PROJECTS</h1>
+			<p class="mb-5">Our projects focus on impacting users and communities by improving their lives with technology. We use various development
+				languages and technologies, while encouraging our contributors to learn new skills and solve new challenges. We host
+				all of our code on GitHub, where contributors can publish and store their code, connect with other open source communities,
+				and learn version control with Git. Github tracks your personal contributions, so you can share them as part of your
+				portfolio.
+			</p>
+		</div>
+		<ngb-tabset [justify]="currentJustify">
+			<ngb-tab title="Peace Corps Projects">
+				<ng-template ngbTabContent>
+					<!-- Peace Corps Projects Note-->
+					<div class="row text-center">
+						<h1 class="mx-auto mt-5 mb-5">PEACE CORPS PROJECTS</h1>
+						<p>The Peace Corps has had nearly 220,000 volunteers (PCVs) in 151 countries around the world since its creation in 1961.
+							Great technology projects are started for the Peace Corps, usually at codeathons, but they are rarely complete enough
+							to deploy to the PCVs. Systers takes on these projects, completes them, then works with Peace Corps administrators
+							to test and deploy in the field.
+							<br>Please note that these projects have been archived until AnitaB.org finds new contacts within the Peace Corps to finish
+							and test them in the field.</p>
+					</div>
+					<!-- Peace Corps Projects Note-->
+					<!-- Projects -->
+					<div class="container box logos mx-auto d-block">
+						<div class="row text-center justify-content-center">
+							<div class="card col-sm-3 mt-3 mb-3 ml-3 mr-3 pt-2" *ngFor="let det of peace_details ;">
+								<img class="card-img-top" src="https://mdbootstrap.com/img/Photos/Lightbox/Thumbnail/img%20(97).jpg" alt="Card image cap">
+								<div class="card-body">
+									<h5 class="card-title">{{det.name}}</h5>
+									<p class="card-text">{{det.summary}}</p>
+									<button type="button" class="btn btn-primary waves-light mx-auto" (click)="style.show()" mdbWavesEffect>View Details</button>
+									<!-- Modal -->
+									<div mdbModal #style="mdb-modal" class="modal fade" id="centralModalSuccess" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
+									    aria-hidden="true">
+										<div class="modal-dialog modal-notify modal-success" role="document">
+											<div class="modal-content">
+												<div class="modal-header" style="background-color: #8d2165;">
```

Comment:
move to css

Reply:
@ancyp Same problem as in the other PR.
I have shifted the style attribute from the paragraph tag to the scss file, but for the model-header class, the style property for background colour when put in the scss file, is overridden by the background-color property (green) of model-header class in the mdBootstrap stylesheet file, since perhaps the newcomers.component.scss file is accessed before the the mdBootstrap default css file and thus the green colour replaces the maroon one mentioned in the css file.
In the case that I add the same statement in the style attribute, it overrides all the existing scss imported attributes and thus the modal-header appears maroon. So in the case we want to remove the style attribute of the modal-header, one would need to go into the depth and analyse how Angular is loading its files. So can we do with the current commit or can we create a separate issue to solve this one, later?